epoxy-adapter/src/main/java/com/airbnb/epoxy/GroupModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.airbnb.epoxy
+
+/**
+ * An [EpoxyModelGroup] usable in DSL.
```

Comment:
could be nice to give a small example of usage (or we can link to the wiki docs here later)

Reply:
Ok, also wdyt to add the `ModelCollector` interface / `add` method in this small class instead of in `EpoxyModelGroup` class, like : 

```
@EpoxyModelClass
abstract class GroupModel : EpoxyModelGroup(), ModelCollector {

    override fun add(model: EpoxyModel<*>) {
        super.addModel(model)
    }
}
```

Because in any case `EpoxyModelGroup` can't be used directly in the DSL. It allow to separate responsibilities.